lib/post/widgets/comment_header.dart

```diff
@@ -44,16 +46,28 @@ class CommentHeader extends StatelessWidget {
                 ),
                 const SizedBox(width: 8.0),
                 Icon(
-                  myVote == VoteType.down ? Icons.south_rounded : Icons.north_rounded,
-                  size: 12.0 * state.contentFontSizeScale.textScaleFactor,
```

Comment:
I would add this line back in to allow to text to scale based on the `textScaleFactor` set in the settings!

`size: 12.0 * state.contentFontSizeScale.textScaleFactor,`